Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/BearerTokenIdentityResolvers/SSOBearerTokenIdentityResolver.swift

```diff
@@ -8,87 +8,296 @@
 import protocol SmithyIdentity.BearerTokenIdentityResolver
```

Comment:
SSO token resolver. The SSO token must have been previously created & cached by the user via another tool like AWS CLI. Then, this token resolver reads the cached token and refreshes it if needed.